src/app/issure/get-records/get-records.component.ts

```diff
@@ -48,6 +53,16 @@ this.getRecords();
     this.generalService.postData('/' + this.documentName + '/search', payout).subscribe((res) => {
     console.log(res);
     this.recordItems = res;
+     for(let i=0;i<this.recordItems.length;i++){
+      //  console.log(this.recordItems[i].contact)
+       this.contactArr[i] = this.recordItems[i].contact;
+       this.newArrOne[i] =  this.contactArr[i].substring(0,6);
+       this.newArrTwo[i] =  this.contactArr[i].substring(6,10);
```

Comment:
@chaitrali-r Create Pipe for mask. I should work for all values like mobile number, email id. abha no. , aadhar etc
- check this one - [link](https://embed.plnkr.co/plunk/KjtLF8)

eg- {{ phoneNumber | mask }}